# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/epplets-extra16/e-dial.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- E-Dial/COMPILE.orig-rock-maint	2003-08-29 08:39:04.000000000 +0200
+++ E-Dial/COMPILE	2003-08-29 09:03:36.000000000 +0200
@@ -2,7 +2,7 @@
 # Run this to install E-Dial on your system.
 
 NAME=E-Dial
-cc edial.c epplet.c -o $NAME.epplet -O2 `imlib-config --libs --cflags`
+$CC $CFLAGS edial.c epplet.c -o $NAME.epplet -O2 `imlib-config --libs --cflags` -L$E16/lib -lepplet -lGL
 cp -f $NAME.epplet $EBIN/$NAME.epplet
 chmod a+x $EBIN/$NAME.epplet
 
